Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/README.md

```diff
@@ -0,0 +1,1910 @@
+                        "type": "application"
+                    },
+                    {
+                        "deprecated": true,
+                        "id": 25,
+                        "integration": false,
+                        "label": "abc/efgh",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": true,
+                        "id": 32,
+                        "integration": false,
+                        "label": "Business Services",
+                        "type": "content"
+                    }
+                ],
+                "count": 1286,
+                "counts": {
+                    "allowedrequests": 1286,
+                    "blockedrequests": 0,
+                    "requests": 1286
+                },
+                "domain": "www.cisco.com",
+                "policycategories": [],
+                "rank": 1
+            },
+            {
+                "bandwidth": null,
+                "categories": [
+                    {
+                        "deprecated": false,
+                        "id": 163,
+                        "integration": false,
+                        "label": "Business and Industry",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": false,
+                        "id": 167,
+                        "integration": false,
+                        "label": "Computers and Internet",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": false,
+                        "id": 142,
+                        "integration": false,
+                        "label": "Online Meetings",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": false,
+                        "id": 148,
+                        "integration": false,
+                        "label": "Application",
+                        "type": "application"
+                    },
+                    {
+                        "deprecated": true,
+                        "id": 25,
+                        "integration": false,
+                        "label": "abc/efgh",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": true,
+                        "id": 32,
+                        "integration": false,
+                        "label": "Business Services",
+                        "type": "content"
+                    }
+                ],
+                "count": 1003,
+                "counts": {
+                    "allowedrequests": 1003,
+                    "blockedrequests": 0,
+                    "requests": 1003
+                },
+                "domain": "presence.teams.microsoft.com",
```

Comment:
```suggestion
                "domain": "dummy.domain.com",
```
please apply to all domains in the readme file.

Reply:
@ShacharKidor modified